src/openrct2-ui/windows/Ride.cpp

```diff
@@ -4709,47 +4709,47 @@ static void WindowRideColourInvalidate(rct_window* w)
         vehicleColour = ride_get_vehicle_colour(ride, w->vehicleIndex);
 
         window_ride_colour_widgets[WIDX_VEHICLE_PREVIEW].type = WindowWidgetType::Scroll;
-        window_ride_colour_widgets[WIDX_VEHICLE_MAIN_COLOUR].type = WindowWidgetType::ColourBtn;
-        window_ride_colour_widgets[WIDX_VEHICLE_MAIN_COLOUR].image = WindowRideGetColourButtonImage(vehicleColour.main);
+        window_ride_colour_widgets[WIDX_VEHICLE_BODY_COLOR].type = WindowWidgetType::ColourBtn;
+        window_ride_colour_widgets[WIDX_VEHICLE_BODY_COLOR].image = WindowRideGetColourButtonImage(vehicleColour.Body);
 
-        bool allowChangingAdditionalColour1 = false;
-        bool allowChangingAdditionalColour2 = false;
+        bool allowChangingTrimColour = false;
+        bool allowChangingTernaryColour = false;
 
         for (int32_t i = 0; i < ride->num_cars_per_train; i++)
         {
             uint8_t vehicleTypeIndex = ride_entry_get_vehicle_at_position(ride->subtype, ride->num_cars_per_train, i);
 
             if (rideEntry->vehicles[vehicleTypeIndex].flags & VEHICLE_ENTRY_FLAG_ENABLE_ADDITIONAL_COLOUR_1)
             {
-                allowChangingAdditionalColour1 = true;
+                allowChangingTrimColour = true;
```

Comment:
You could also rename the VEHICLE_ENTRY_FLAG_ENABLE_ADDITIONAL_COLOUR_1 whilst you are at it

Reply:
Missed that! I'll rename it, thanks.

Reply:
Done. Renamed both VEHICLE_ENTRY_FLAG_ENABLE_ADDITIONAL_COLOUR_1 and VEHICLE_ENTRY_FLAG_ENABLE_ADDITIONAL_COLOUR_2.